wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -2377,6 +2738,22 @@
 			<wsdl:input message="tas:CreateRSAKeyPairRequest"/>
 			<wsdl:output message="tas:CreateRSAKeyPairResponse"/>
 		</wsdl:operation>
+		<wsdl:operation name="CreateECCKeyPair">
+			<wsdl:documentation>
+				This operation triggers the asynchronous generation of an ECC key pair using a particular elliptic curve as specified in [RFC 4492], with a suitable key generation mechanism on the device.
+				Keys, especially RSA key pairs, are uniquely identified using key IDs.<br/>
```

Comment:
especially RSA key pairs, are uniquely identified using key IDs.  is this ECC Key pair?